CHANGES.rst

```diff
@@ -1,6 +1,27 @@
 Changes
 =======
 
+0.5.3 (2024-MM-DD)
```

Comment:
I think that with new features we should be increasing the minor version, not patch version.

Reply:
Let's leave patch version increases for bug fixes only.